.github/workflows/nightly.yml

```diff
@@ -189,7 +189,6 @@ jobs:
             python-jsonschema:p
             curl:p
             drmingw:p
-            exiv2:p
```

Comment:
I think we can leave this (could be a dependency of other stuff and installed anyway), the '--noconfirm' below should downgrade silently.

Looks good otherwise, thanks.

Reply:
I checked by running an action in my fork, no, exiv2 is not installed as a dependency of other stuff. It's good that `--noconfirm` also allows downgrading, but I don't see the point of installing a package to downgrade it right away.

Reply:
Just one less thing to forget about when restoring back some day 😉

Reply:
I'm hoping that when I'll remove the temporary installation of the older package I won't be suffering from senile dementia to the point where I immediately forget to add exiv2 to the main list. 😉

Reply:
I should hope so 😉 I was just speaking from my perspective, can't remember everything I did yesterday... 😮